src/commands/utils/monitor-control/set_brightness.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+
+. ./utility_functions.sh
```

Comment:
```suggestion
. ../../common-script.sh
```
regular . ./ won't work.

Reply:
why are we changing the importing of functions which are in utility function file to common-script?